pkgs/build-support/rust/default.nix

```diff
@@ -36,21 +37,22 @@ let
       cargoDepsCopy="$sourceRoot/${cargoVendorDir}"
     '';
 
+  hostCC="${buildPackages.stdenv.cc}/bin/${buildPackages.stdenv.cc.targetPrefix}cc";
+  hostCXX="${buildPackages.stdenv.cc}/bin/${buildPackages.stdenv.cc.targetPrefix}cxx";
+  targetCC="${stdenv.cc.targetPrefix}cc";
+  targetCXX="${stdenv.cc.targetPrefix}cxx";
```

Comment:
These should be called "build" and "host", not "host" and "target".

Reply:
FWIW I personally like the autoconf-esque `ccForBuild` `ccForHost`, etc. The "for" emphasizes we're talking about what the compilers target, not where they run.